v3/integration/lints/lints/init_first_test.go

```diff
@@ -0,0 +1,47 @@
+package lints
+
+/*
+ * ZLint Copyright 2021 Regents of the University of Michigan
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import (
+	"os"
+	"testing"
+
+	"github.com/zmap/zlint/v3/integration/lints/lint"
+)
+
+func TestInitFirst_Lint(t *testing.T) {
+	t.Log(os.Getwd())
+	data := map[string]bool{
+		"testdata/initializeFirst.go":            true,
+		"testdata/initializeNotFirst.go":         false,
+		"testdata/initializeFirstNoFunctions.go": false,
+	}
+	l := &InitFirst{}
+	for file, want := range data {
+		t.Run(file, func(t *testing.T) {
+			r, err := lint.RunLint(file, l)
+			if err != nil {
+				t.Error(err)
+				return
```

Comment:
This feels like a place I'd be tempted to use `t.Fatal` instead of `t.Error` + `return`.

Reply:
Would fatal panic that the parent cause the other tests to not be attempted?

Edit: No it does not, `t.Run` appears to recover.

Reply:
Yup! It should be safe to use in this context.